sbt-plugin/src/main/scala/LanguageFeaturesPlugin.scala

```diff
@@ -0,0 +1,161 @@
+import featuressetcomputer.FeaturesSetComputer
+
+import sbt._
+import Keys._
+import java.io.File
+import scala.meta.Dialect
+
+object LanguageFeaturesPlugin extends AutoPlugin {
+
+  override def trigger = allRequirements
+
+  object autoImport {
+    val languageFeaturesConfig =
+      settingKey[LanguageFeaturesConfig]("Configuration of the checker.")
+    val languageFeaturesCheck = taskKey[Unit]("Check language features.")
+
+    @inline final val LanguageFeaturesConfig =
+      sbtlanguagefeatures.LanguageFeaturesConfig
+
+    import scala.meta.dialects
+    @inline final val Scala3 = dialects.Scala3
+    @inline final val Scala213 = dialects.Scala213
+    @inline final val Scala212 = dialects.Scala212
+
+    import syntactic.blacklist
+    @inline final val BlacklistChecker = blacklist.BlacklistChecker
+
+    import blacklist.PredefBlacklistRules
+    @inline final val NoCast = PredefBlacklistRules.NoCast
+    @inline final val NoNull = PredefBlacklistRules.NoNull
+    @inline final val NoVar = PredefBlacklistRules.NoVar
+    @inline final val NoWhile = PredefBlacklistRules.NoWhile
+
+    import syntactic.whitelist
+    @inline final val WhitelistChecker = whitelist.WhitelistChecker
+
+    import whitelist.PredefFeatures
+    @inline final val LiteralsAndExpressions =
+      PredefFeatures.LiteralsAndExpressions
+    @inline final val Nulls = PredefFeatures.Nulls
+    @inline final val Vals = PredefFeatures.Vals
+    @inline final val Defs = PredefFeatures.Defs
+    @inline final val ADTs = PredefFeatures.ADTs
+    @inline final val LiteralFunctions = PredefFeatures.LiteralFunctions
+    @inline final val ForExpr = PredefFeatures.ForExpr
+    @inline final val PolymorphicTypes = PredefFeatures.PolymorphicTypes
+    @inline final val Laziness = PredefFeatures.Laziness
+    @inline final val BasicOop = PredefFeatures.BasicOop
+    @inline final val AdvancedOop = PredefFeatures.AdvancedOop
+    @inline final val ImperativeConstructs = PredefFeatures.ImperativeConstructs
+    @inline final val ContextualConstructs = PredefFeatures.ContextualConstructs
+    @inline final val Extensions = PredefFeatures.Extensions
+    @inline final val Metaprogramming = PredefFeatures.Metaprogramming
+    @inline final val Packages = PredefFeatures.Packages
+    @inline final val Imports = PredefFeatures.Imports
+    @inline final val Exports = PredefFeatures.Exports
+    @inline final val Xml = PredefFeatures.Xml
+    @inline final val StringInterpolation = PredefFeatures.StringInterpolation
+    @inline final val Annotations = PredefFeatures.Annotations
+    @inline final val Infixes = PredefFeatures.Infixes
+    @inline final val Inlines = PredefFeatures.Inlines
+  }
+
+  import autoImport._
+
+  // Get files with .scala extension under the directory
+  private def getScalaFiles(dir: File): List[File] = {
+    val files = dir.listFiles.toList
+    files.filter { file =>
+      file.getName.split("\\.").lastOption match {
```

Comment:
FYI, you could also write `file.endsWith(".scala")`

Reply:
Modified in [beb6711](https://github.com/civp/language-features-a-la-carte/pull/9/commits/beb67113612b3126cd3deea11781c5cb476b9f0d)